hummingbot/connector/derivative/dydx_perpetual/dydx_perpetual_utils.py

```diff
@@ -0,0 +1,51 @@
+from hummingbot.client.config.config_var import ConfigVar
+from hummingbot.client.config.config_methods import using_exchange
+
+
+CENTRALIZED = True
+
+
+EXAMPLE_PAIR = "BTC-USD"
+
+
+DEFAULT_FEES = [0.02, 0.04]
```

Comment:
```suggestion
DEFAULT_FEES = [0.05, 0.2]
```
According to the website, I see that's the maker-taker fee percent.

Reply:
Resolved in https://github.com/CoinAlpha/hummingbot/pull/3130/commits/c409bd3e9840b47e73eefa42d5264e7b7d6e87fd